server/src/main/java/io/crate/metadata/doc/DocTableInfo.java

```diff
@@ -709,25 +709,23 @@ public Collection<ColumnIdent> notNullColumns() {
     }
 
     /**
-     * Starting from 5.5 column OID-s are used as source keys.
-     * Even of 5.5, there are no OIDs (and thus no source key rewrite happening) for:
-     * <ul>
-     *  <li>OBJECT (IGNORED) sub-columns</li>
-     *  <li>Internal object keys of the geo shape column, such as "coordinates", "type"</li>
-     * </ul>
+     * For tables >= 5.5 this returns a function that takes a oid and returns the corresponding column name or null if missing or dropped.
```

Comment:
Actually, the given `oidOrName` is returned if missing (to account for `ignored` cols afaik).
```suggestion
     * For tables >= 5.5 this returns a function that takes a oid and returns the corresponding column name or null if dropped.
```